src/main/java/org/broadinstitute/hellbender/tools/copynumber/utils/germlinetagging/SimpleGermlineTagger.java

```diff
@@ -65,31 +78,39 @@ private SimpleGermlineTagger(){}
                 .collect(Collectors.toList());
 
         final Map<AnnotatedInterval, List<AnnotatedInterval>> nonZeroMergedNormalSegmentsToTumorSegments = IntervalUtils.createOverlapMap(nonZeroMergedNormalSegments, tumorSegments, dictionary);
-        final Map<AnnotatedInterval, CalledCopyRatioSegment.Call> tumorSegsToGermlineTag = createTumorSegmentsToGermlineTagMap(nonZeroMergedNormalSegmentsToTumorSegments, paddingInBp, callAnnotation);
+        final Map<AnnotatedInterval, CalledCopyRatioSegment.Call> tumorSegsToGermlineTag = createTumorSegmentsToGermlineTagMap(nonZeroMergedNormalSegmentsToTumorSegments, paddingInBp, callAnnotation, reciprocalThreshold);
 
         return tumorSegments.stream()
                 .map(s -> createTumorTaggedSegment(s, outputAnnotationName, tumorSegsToGermlineTag.getOrDefault(s, CalledCopyRatioSegment.Call.NEUTRAL).getOutputString()))
                 .collect(Collectors.toList());
     }
 
-    private static Map<AnnotatedInterval, CalledCopyRatioSegment.Call> createTumorSegmentsToGermlineTagMap(final Map<AnnotatedInterval, List<AnnotatedInterval>> nonZeroMergedNormalSegmentsToTumorSegments, int paddingInBp, final String callAnnotation) {
+    private static Map<AnnotatedInterval, CalledCopyRatioSegment.Call> createTumorSegmentsToGermlineTagMap(final Map<AnnotatedInterval, List<AnnotatedInterval>> nonZeroMergedNormalSegmentsToTumorSegments, int paddingInBp, final String callAnnotation, final double reciprocalThreshold) {
         final Map<AnnotatedInterval,CalledCopyRatioSegment.Call> result = new HashMap<>();
         for (final AnnotatedInterval normalSeg : nonZeroMergedNormalSegmentsToTumorSegments.keySet()) {
             final List<AnnotatedInterval> overlappingTumorSegments = nonZeroMergedNormalSegmentsToTumorSegments.get(normalSeg);
+
+            final List<AnnotatedInterval> mergedTumorSegments = mergedRegionsByAnnotation(callAnnotation, overlappingTumorSegments);
+            final boolean isReciprocolOverlapSeen = mergedTumorSegments.stream()
+                .anyMatch(s -> (normalSeg.getInterval().intersect(s).size() > (s.getInterval().size() * reciprocalThreshold)) &&
```

Comment:
Probably cleaner to extract a test for reciprocal overlap, but up to you.

Reply:
Done... Added utility to `IntervalUtils`